src/components/SearchCard.tsx

```diff
@@ -0,0 +1,70 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import {
+  flexCenter,
+  transformScale,
+  textOverflowEllipsis,
+} from '@/styles/theme';
+import { IMovie } from '@/interfaces/interfaces';
+import { IMAGE_BASE_URL } from '@/utils/constants';
+
+interface SearchCardProps {
+  movie: IMovie;
+}
+
+const SearchCard = ({ movie }: SearchCardProps) => {
+  return (
+    <Link href={`detail/${movie.id}`}>
+      <div className="wrapper">
+        <Image
+          src={`${IMAGE_BASE_URL}/${movie.poster_path}`}
+          width={146}
+          height={76}
+          alt="image"
+          style={{ objectFit: 'cover' }}
+        />
```

Comment:
![image](https://github.com/CEOS-Developers/next-netflix-17th/assets/15906101/c0707418-ea2b-4645-84e7-173019f62b64)
에서처럼 image가 없는 경우에 Netflix 로고를 띄워두거나 하면 좋은 것 같아요!
```suggestion
        {movie.poster_path === null ? (
            <Image
              src="/netflix-logo.png"
              width={146}
              height={76}
              alt={movie.title}
              style={{ objectFit: 'cover' }}
            />
          ) : (
            <Image
              src={`${IMAGE_BASE_URL}/${movie.poster_path}`}
              width={146}
              height={76}
              alt={movie.title}
              style={{ objectFit: 'cover' }}
            />
          )}
```